### PR TITLE
Feature/view post screen

### PR DESCRIPTION
**Pull request by WalterGoldenOwl**
Description Pull Request:
## Todo
- Code UI and logic view full post (like, comment, ...)

## Task
Link of task: 

1. https://trello.com/c/f4vLQ8hW
2. https://trello.com/c/MszSEVqN

## Screenshot
Screenshot_1697336360
Screenshot_1697336369

## Video

https://github.com/dustin-goldenowl/jobspot/assets/144195318/8ee6842e-dd4b-4e4e-802d-e04ce00f002f

### CODE REVIEW

File: lib/src/core/common/widgets/custom_alert_dialog.dart [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    ),
+    actions: [
+      IntrinsicHeight(
+        child: Row(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          children: [
+            Expanded(
+              child: CustomButton(
+                onPressed: () => context.router.pop(),
+                title: "Cancel",
```
Comment (dustin-goldenowl) [inline lib/src/core/common/widgets/custom_alert_dialog.dart]: using app_en.arb

File: lib/src/core/common/widgets/custom_alert_dialog.dart [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+              ),
+            ),
+            const SizedBox(width: 15),
+            Expanded(
+              child: CustomButton(
+                isElevated: false,
+                onPressed: onOK,
+                title: "OK",
```
Comment (dustin-goldenowl) [inline lib/src/core/common/widgets/custom_alert_dialog.dart]: using app_en.arb

File: lib/src/core/common/widgets/custom_alert_dialog.dart (hunk 1 above)
Comment (WalterGoldenOwl) [reply lib/src/core/common/widgets/custom_alert_dialog.dart]: I fixed it

File: lib/src/core/common/widgets/custom_alert_dialog.dart (hunk 2 above)
Comment (WalterGoldenOwl) [reply lib/src/core/common/widgets/custom_alert_dialog.dart]: I fixed it

**Pull request merged**